python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -450,8 +450,8 @@ def explain_node(
         -------
         Tensor
             Classification probabilities given the masked graph. It is a tensor of
-            shape :math:`(B, L)`, where :math:`L` is the different types of label
-            in the dataset, and :math:`B` is the batch size.
+            shape :math:`(B, N)`, where :math:`L` is the different types of node
```

Comment:
Is this shape still correct?  Where is `L`?

Reply:
@ndbaker1 can you confirm.

Reply:
corrected to `(N, L)`, just forgot to update that doc, thanks for the catch 🐟